packages/apps/spaces/app/invoices/src/components/InvoicesTable/Columns/Headers/Avatar/AvatarHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import { useFeatureIsOn } from '@growthbook/growthbook-react';
+
+import { Flex } from '@ui/layout/Flex';
+import { Plus } from '@ui/media/icons/Plus';
+import { Tooltip } from '@ui/overlay/Tooltip';
+import { IconButton } from '@ui/form/IconButton';
+import { useOrganizationsPageMethods } from '@organizations/hooks/useOrganizationsPageMethods';
+
+export const AvatarHeader = () => {
+  const enableFeature = useFeatureIsOn('gp-dedicated-1');
+  const { createOrganization } = useOrganizationsPageMethods();
+
+  const handleCreateOrganization = () => {
+    createOrganization.mutate({ input: { name: '' } });
+  };
+
+  return (
+    <Flex w='42px' align='center' justify='center'>
+      <Tooltip
+        label='Create an organization'
+        visibility={enableFeature ? 'visible' : 'hidden'}
+      >
+        <IconButton
+          size='sm'
+          variant='ghost'
+          aria-label='create organization'
+          onClick={handleCreateOrganization}
+          isLoading={createOrganization.isPending}
+          icon={<Plus color='gray.400' boxSize='5' />}
+          visibility={enableFeature ? 'visible' : 'hidden'}
+        />
+      </Tooltip>
+    </Flex>
+  );
+};
```

Comment:
If this is intentional or you plan on changing later then all good. This component I think is not used in the invoices table.